files/en-us/web/api/web_share_api/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: Web Share API
+slug: Web/API/Web_Share_API
+tags:
+  - API
+  - Apps
+  - Web Share
+  - Web Share API
+  - Guide
+  - Overview
+---
+<div>{{DefaultAPISidebar("Web Share API")}}</div>
+
+<p>The <strong>Web Share API</strong> provides a mechanism for sharing text, links, files and other content to an arbitrary <em>share target</em> selected by the user.</p>
+
+<p>The available share targets are provided using the native mechanisms of the underlying operating system. They might, for example, include the system clipboard or other services, Bluetooth or WiFi, email, contacts or messaging applications, and websites.</p>
+
+<p>The API consists of just two methods:</p>
+
+<ul>
+  <li>{{domxref("navigator.canShare()")}}: Validate that the specified data is sharable (check if a call with the same data to {{domxref("navigator.share()")}} would succeed).</li>
+  <li>{{domxref("navigator.share()")}}: Share the specified data. This method must be called on a button click or other user activation (requires {{Glossary("transient activation")}}).</li>
```

Comment:
```suggestion
  <li>{{domxref("navigator.share()")}}: Returns a {{jsxref("Promise")}} that resolves if the passed data was successfully shared. This method must be called on a button click or other user activation (requires {{Glossary("transient activation")}}).</li>
```

Reply:
@jpmedley Re "resolves if the passed data was successfully shared". Technically if "successfully passed to a share target", since it can fail before actually being shared outside of this API. I have modified but let me know if you think the distinction is not useful. 

```suggestion
  <li>{{domxref("navigator.share()")}}: Returns a {{jsxref("Promise")}} that resolves if the passed data was successfully sent to a share target. This method must be called on a button click or other user activation (requires {{Glossary("transient activation")}}).</li>
```